README.md

```diff
@@ -22,6 +22,8 @@ Python wheels are great. Building them across **Mac, Linux, Windows**, on **mult
 What does it do?
 ----------------
 
+While cibuildwheel itself requires a recent enough python version to run, it can target the following versions to build wheels:
```

Comment:
```suggestion
While cibuildwheel itself requires a recent Python version to run (we support the last three releases), it can target the following versions to build wheels:
```